src/main/java/com/icthh/xm/ms/entity/domain/XmEntity.java

```diff
@@ -114,7 +111,7 @@
     @javax.validation.constraints.NotNull
     @ApiModelProperty(value = "Key reference to the configured Entity Type. Convention is capital letter with dash '-'. Example: ACCOUNT, PRODUCT-OFFERING, PRICE etc. Entity Sub Types could be separated by dot '.'. Convention is same as for Type. Example: ACCOUNT.ADMIN, ACCOUNT.USER, ACCOUNT.PARTNER for type ACCOUNT or PRODUCT-OFFERING.CAMPAIGN, PRODUCT-OFFERING.RATE-PLAN etc.", required = true)
     @Column(name = "type_key", nullable = false)
-    @Field(index = FieldIndex.not_analyzed, type = FieldType.String)
+    @Field(type = FieldType.Keyword)
```

Comment:
same as prev question

Reply:
https://stackoverflow.com/questions/45406577/how-to-config-not-analyzed-with-field-annotation-in-spring-data-elasticsearch-3